client_lib/test/unit/client/extensions/test_exceptions.py

```diff
@@ -416,3 +416,22 @@ def test_certificate_expiration(self):
 
         # Verify
         self.assertEqual(date, 'May  9 12:39:37 2013 GMT')
+
```

Comment:
I would like to see these tests use the Mock library to so you can test all the options. That will also not depend on the state of logging.NOTSET.